compiler/aarch64/codegen/OMRCodeGenerator.hpp

```diff
@@ -302,6 +302,9 @@ class OMR_EXTENSIBLE CodeGenerator : public OMR::CodeGenerator
    // OutOfLineCodeSection List functions
    TR::list<TR_ARM64OutOfLineCodeSection*> &getARM64OutOfLineCodeSectionList() {return _outOfLineCodeSectionList;}
 
+   int32_t arrayTranslateMinimumNumberOfElements(bool isByteSource, bool isByteTarget) { return 8; } //FIXME
+   int32_t arrayTranslateAndTestMinimumNumberOfIterations() { return 8; } //FIXME
```

Comment:
Rather than "FIXME", please reference issue #4446 that I created for this.

Reply:
Removed "FIXME" and added comment with the reference to issue #4446.